### DISCUSSION TITLE
numerous contributions but dull green

### DISCUSSION THREAD

**Discussion by p3rc1us** [community/community#119942] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/144686514/d53fc507-d6e5-4b13-b63a-75b35814cf23)

Hello, i have numerous contributions each day, this year. And the green highlights in my chart must be vibrant. but as you can see, the color of it is like I just contributed 1 or 2 commits. My chart is doing Okay 2 days ago but now it is almost uniformly low contribution color. thanks!

**Comment by MindOfBear** [comment]
More contributions you have on **main** brach, the more greener the chart should be, try pulling on the main branch the commits from other branches, that should make your chart greener.

**Reply by p3rc1us** [reply]
![image](https://github.com/community/community/assets/144686514/fb4bb59f-a8de-4076-be05-09d33a86bf87)

I see, but the first time I work with branches is on April 2nd. The rest of it (since January) I only work in Main. That's why earlier this week this chart is just doing fine. greener. but it became like this, idk why :(
